specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/examples/ViewRender.json

```diff
@@ -0,0 +1,75 @@
+{
+  "parameters": {
+    "api-version": "2019-11-01",
+    "scope": "subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MYDEVTESTRG",
+    "parameters": {
+      "eTag": "\"1d4ff9fe66f1d10\"",
+      "properties": {
+        "displayName": "swagger Example",
+        "query": {
+          "type": "Usage",
+          "timeframe": "MonthToDate",
+          "dataset": {
+            "granularity": "Daily",
+            "aggregation": {
+              "totalCost": {
+                "name": "PreTaxCost",
+                "function": "Sum"
+              }
+            },
+            "grouping": [],
+            "sorting": [
+              {
+                "direction": "Ascending",
+                "name": "UsageDate"
+              }
+            ]
+          }
+        },
+        "chart": "Table",
+        "accumulated": "true",
+        "metric": "ActualCost",
+        "kpis": [
+          {
+            "type": "Forecast",
+            "id": null,
+            "enabled": true
+          },
+          {
+            "type": "Budget",
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MYDEVTESTRG/providers/Microsoft.Consumption/budgets/swaggerDemo",
+            "enabled": true
+          }
+        ],
+        "pivots": [
+          {
+            "type": "Dimension",
+            "name": "ServiceName"
+          },
+          {
+            "type": "Dimension",
+            "name": "MeterCategory"
+          },
+          {
+            "type": "TagKey",
+            "name": "swaggerTagKey"
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MYDEVTESTRG/providers/Microsoft.CostManagement/views/swaggerExample",
```

Comment:
is this POST action creating a new resource in its current form?  What does this ID point to?  Why is the resource type "downloadLink"?

Reply:
Change reverted